framework/test/gtest/BundleHooksTest.cpp

```diff
@@ -209,30 +232,32 @@ void TestEventHookFailure(const Framework& framework)
   bundleA.Start();
 
   // bundle starting and bundle started events
-  US_TEST_CONDITION_REQUIRED(listener.events.size() == 2,
-                             "Test for expected number of framework events");
-
-  std::for_each(
-    listener.events.begin(),
-    listener.events.end(),
-    [](const FrameworkEvent& evt) {
-      US_TEST_CONDITION_REQUIRED(evt.GetThrowable() != nullptr,
-                                 "Test for the existence of an exception");
-      US_TEST_CONDITION_REQUIRED(evt.GetType() ==
-                                   FrameworkEvent::Type::FRAMEWORK_WARNING,
-                                 "Test for the correct framework event type");
-      std::string msg(evt.GetMessage());
-      US_TEST_CONDITION_REQUIRED(
-        std::string::npos != msg.find("Failed to call Bundle EventHook #"),
-        "Test for the correct event message");
-    });
+#ifdef US_BUILD_SHARED_LIBS
+  ASSERT_EQ(listener.events.size(), 2);
+#else
+  ASSERT_GE(static_cast<int>(listener.events.size()), 2);
+#endif
```

Comment:
what is the # of framework events returned under a static build configuration?

Reply:
It is 3 for all the builds.
I have confirmed that with [appveyor build](https://ci.appveyor.com/project/apatil19/cppmicroservices-m34e5)